crates/turborepo-lib/src/cli.rs

```diff
@@ -461,8 +461,20 @@ pub async fn run(repo_state: Option<RepoState>) -> Result<Payload> {
 
             Ok(Payload::Rust(Ok(0)))
         }
-        Command::Link { .. }
-        | Command::Unlink { .. }
+        Command::Link { no_gitignore } => {
+            if clap_args.test_run {
+                println!("Link test run successful");
+                return Ok(Payload::Rust(Ok(0)));
+            }
+
+            let modify_gitignore = !*no_gitignore;
+            let base = CommandBase::new(clap_args, repo_root)?;
+
+            link::link(base, modify_gitignore).await?;
```

Comment:
I think we should be logging the error here before returning it to match the Go impl